virtaal/controllers/unitcontroller.py

```diff
@@ -123,7 +125,7 @@ def load_unit(self, unit):
             #        because the names could have changed in the new document :/
             state_names = self.get_unit_state_names()
             if state_names:
-                unit._workflow = workflow.create_unit_workflow(unit, state_names)
+                unit._workflow = create_unit_workflow(unit, state_names)
```

Comment:
> I must be missing something, but why is this required?

I don't know what it's doing, just trying to fix the error while keeping the original code

Reply:
Ok, I see now that the code is copied from the Translate Toolkit. It isn't compatible with Python 3, so that is why you made the local copy in Virtaal

Reply:
> Ok, I see now that the code is copied from the Translate Toolkit. It isn't compatible with Python 3, so that is why you made the local copy in Virtaal

I know this is bad behavior for code management.But I was too impatient to wait until the Translate Toolkit fixed it.As official code, it should wait for Translate Toolkit to fix it